src/fsharp/PackageManager.Integration.fs

```diff
@@ -38,15 +38,17 @@ type PackageManager = {
     ToolName: string
     Name:string }
 
-let RegisteredPackageManagers : PackageManager list = 
+let RegisteredPackageManagers = lazy (
```

Comment:
"register some handlers" 

the question is: how do we do that. My assumption was it needs to find some dlls somewhere.
Then a single eval would be enough, right?